packages/next-auth/src/providers/saml-jackson.ts

```diff
@@ -0,0 +1,37 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export interface SAMLJacksonProfile {
+  id: string;
+  email: string;
+  firstName: string | null | undefined;
+  lastName: string | null | undefined;
```

Comment:
```suggestion
  email?: string;
  firstName?: string | null;
  lastName?: string | null;
```

Let's prefer optional parameters.